cc-base-app/main/default/lwc/experienceSchedule/experienceSchedule.js

```diff
@@ -1,8 +1,6 @@
 import { LightningElement, api, wire } from 'lwc';
 import LightningAlert from 'lightning/alert';
 import { NavigationMixin } from 'lightning/navigation';
-import { subscribe, MessageContext } from 'lightning/messageService';
-import EXPERIENCE_SELECTED_MESSAGE from '@salesforce/messageChannel/ExperienceSelected__c';
```

Comment:
Why is this removed?

Reply:
I had to combine the experience list and the schedule components into a single component because of how Aura sites don’t deal well with split sections on their home page :grin:
Since both component share ancestry, I’ve used the recordId API attribute to pass the selection. This way it’s consistent with what we have in the Org.